raiden/network/rpc/smartcontract_proxy.py

```diff
@@ -76,6 +80,19 @@ def broadcast_signed_transaction(self, signed_tx):
         txhash = self.jsonrpc_client.send_signed_transaction(signed_tx)
         return txhash
 
+    def broadcast_signed_transaction_and_wait(self, signed_tx):
+        try:
+            transaction_hash = self.broadcast_signed_transaction(signed_tx)
+            self.jsonrpc_client.poll(transaction_hash)
+            receipt = check_transaction_threw(self.jsonrpc_client, transaction_hash)
+            if receipt:
+                raise RaidenRecoverableError("transaction failed")
+        except HTTPError:
+            raise RawTransactionFailed("Transaction malformed")
+        except Exception as e:
+            raise e
```

Comment:
what is the point of this?

Reply:
this code is copy pasted everywhere, I just made a function to prevent doing this again, and to refactor someday. we could make an issue for that